src/main/resources/application.properties

```diff
@@ -38,4 +38,7 @@ out/
 src/main/java/com/coolbank/Log/*
 src/main/resources/*
 src/main/resources/application.properties
-src/main/resources/data.sql
\ No newline at end of file
+src/main/resources/data.sql
+##############################
+## PostgreSQL DB
+##############################
```

Comment:
?)

Reply:
I started to take place for my connections